### PR TITLE
Feature/us5517 share a public group

### PR DESCRIPTION
**Pull request by ilpadre**

### CODE REVIEW

File: Gateway/crds-angular/Controllers/API/GroupToolController.cs [hunk 1]
```diff
@@ -47,18 +47,18 @@ public class GroupToolController : MPAuth
...
-                    var invitess = _groupToolService.GetInvitations(sourceId, invitationTypeId, token);
-                    return Ok(invitess);
-                }
-                catch (Exception exception)
-                {
-                    var apiError = new ApiErrorDto("GetInvitations Failed", exception);
-                    throw new HttpResponseException(apiError.HttpResponseMessage);
-                }
-            });
+                              {
```
Comment (Doug-Shannon) [inline Gateway/crds-angular/Controllers/API/GroupToolController.cs]: Whats up with the weird spacing all throughout this file?

File: crossroads.net/app/group_tool/groupTool.routes.js [hunk 2]
```diff
@@ -200,7 +201,7 @@ export default function GroupToolRouter($httpProvider, $stateProvider) {
     })
     .state('grouptool.search', {
       parent: 'noSideBar',
-      url: '/groups/search?age&category&type&kids&grouplocation&day&time&frequency&site',
+      url: '/groups/search?age&category&type&kids&grouplocation&day&time&frequency&site&id',
```
Comment (Doug-Shannon) [inline crossroads.net/app/group_tool/groupTool.routes.js]: not sure ID needs to be added as a param to the search page.. Should only be needed on the results page.  Cant imagine a reason for passing someone with an ID to the page for them to add additional search parameters.

File: crossroads.net/app/group_tool/group_detail/about/groupDetail.about.component.js [hunk 3]
```diff
@@ -4,21 +4,21 @@ GroupDetailAboutComponent.$inject = [ ];
...
       forInvitation: '<',
       forSearch: '<',
-      isLeader: '<'
+      isLeader: '<',
+      showShareButtons: '<'
     },
     restrict: 'E',
     templateUrl: 'about/groupDetail.about.html',
-    controller: controller,
+    controller,
```
Comment (Doug-Shannon) [inline crossroads.net/app/group_tool/group_detail/about/groupDetail.about.component.js]: Isn't it more expressive to have controller: controller?  Doesn't really matter though

Comment (JoeKer1) [inline crossroads.net/app/group_tool/group_detail/about/groupDetail.about.component.js]: It is our linting rule to not have it if the names match.

File: Gateway/crds-angular/Controllers/API/GroupToolController.cs (hunk 1 above)
Comment (ilpadre) [inline Gateway/crds-angular/Controllers/API/GroupToolController.cs]: No clue. I reformatted the file and seems to be ok now.

File: crossroads.net/app/group_tool/groupTool.routes.js (hunk 2 above)
Comment (ilpadre) [inline crossroads.net/app/group_tool/groupTool.routes.js]: I think this was a residual of my original figuring out what needed to be done. Has been removed and tests run.

**Pull request merged**